test/module/dy/fns/replace.spec.js

```diff
@@ -0,0 +1,99 @@
+const expect = require('chai').expect;
+const { describe } = require('node-tdd');
+const { LocalTable, buildModel } = require('../../../dy-helper');
+const { ModelNotFound } = require('../../../../src/resources/errors');
+
+describe('Testing replace', {
+  useNock: true,
+  nockStripHeaders: true,
+  envVarsFile: '../../../default.env.yml'
+}, () => {
+  let model;
+  let localTable;
+  let item;
+  let key;
+  let getItemOrNull;
+  let generateItem;
+
+  before(() => {
+    key = {
+      id: '123',
+      name: 'name'
+    };
+  });
+  beforeEach(async () => {
+    model = buildModel();
+    localTable = LocalTable(model);
+    await localTable.create();
+    item = {
+      ...key,
+      age: 50
+    };
+    getItemOrNull = (k) => model.getItem(k, {
+      onNotFound: (i) => null
+    });
+    generateItem = () => model.create(item);
+  });
+  afterEach(async () => {
+    await localTable.delete();
+  });
+
+  it('Testing replace item replaced', async () => {
```

Comment:
in this test you need to check that an additional field gets removed with the replace (opposed to modify where that field would stay)